R/radial_ggplot.R

```diff
@@ -83,7 +85,9 @@ radial_ggplot <- function(polar,
                           colour_code_labels = TRUE,
                           label_colour = NULL,
                           grid_colour = "grey80", 
+                          grid_width = 1,
                           axis_colour = "black",
+                          axis_width = 2,
                           axis_title_size = 5,
```

Comment:
axes_from_origin is specific to polar_grid so can be passed in as part of the ..., doesn't need its own param